cuenca/http/client.py

```diff
@@ -52,7 +52,7 @@ def configure(
             if sandbox:
                 self.base_url = SANDBOX_URL
             else:
-                self.base_url = API_URL
+                self.base_url = API_URL  # pragma: no cover
```

Comment:
use a version of this to test prod: https://github.com/cuenca-mx/arcus-python/blob/master/tests/test_client_auth.py#L22-L27